src/ui/public/agg_types/buckets/terms.js

```diff
@@ -78,7 +80,23 @@ export const termsBucketAgg = new BucketAggType({
     if (aggConfig.params.otherBucket) {
       const filterAgg = buildOtherBucketAgg(aggConfigs, aggConfig, resp);
       nestedSearchSource.set('aggs', filterAgg);
+
+      const request = aggConfigs.vis.API.inspectorAdapters.requests.start('Other Bucket request', {
+        description: `This request counts how much documents aren't included in the
+               actual data to build the "Other" bucket from that information.`
+      });
+      nestedSearchSource.getSearchRequestBody().then(body => {
+        request.json(body);
+      });
+      request.stats(getRequestInspectorStats(nestedSearchSource));
+
       const response = await nestedSearchSource.fetchAsRejectablePromise();
+      // TODO: Remove delay and error before merging
+      setTimeout(() => {
+        request
+          .stats(getResponseInspectorStats(nestedSearchSource, response))
+          .error({ json: response });
```

Comment:
This will be changed to `ok` before merging, but that way we can demonstrate a failure case for reviewing the PR.